src/NUnitFramework/tests/Api/FrameworkControllerTests.cs

```diff
@@ -151,6 +166,22 @@ public void LoadTestsAction_BadFile_ReturnsNonRunnableSuite()
 
         #region ExploreTestsAction
         [Test]
+        public void ExploreTestsAction_FilterCategory_ReturnsTests()
+        {
+            new FrameworkController.LoadTestsAction(_controller, _handler);
+            new FrameworkController.ExploreTestsAction(_controller, FIXTURE_CAT_FILTER, _handler);
+            var result = TNode.FromXml(_handler.GetCallbackResult());
+
+            Assert.That(result.Name.ToString(), Is.EqualTo("test-suite"));
+            Assert.That(result.Attributes["type"], Is.EqualTo("Assembly"));
+            Assert.That(result.Attributes["id"], Is.Not.Null.And.StartWith("ID"));
+            Assert.That(result.Attributes["name"], Is.EqualTo(EXPECTED_NAME));
+            Assert.That(result.Attributes["runstate"], Is.EqualTo("Runnable"));
+            Assert.That(result.Attributes["testcasecount"], Is.EqualTo(MockAssembly.Tests.ToString()));
```

Comment:
Yes - I have been focusing on the nunitlite runner rather that the other plumbing for the framework. Once I have the basic structure done (which I think it pretty much there with minor update), I'll add in the extra items into the mix :smiley: